admin-src/index.js

```diff
@@ -21,13 +21,19 @@ const App = () => {
 
   const onUserLogin = () => setUserLoggedIn(true)
 
+  useEffect(() => {
+    firebase.auth().onAuthStateChanged((user) => {
+      if (user) {
+        onUserLogin()
+      }
+    })
+  })
```

Comment:
This will execute **every time** the `App` component renders. Please add an empty dependencies array so that it's only executed when the component is mounted:
```suggestion
  useEffect(() => {
    firebase.auth().onAuthStateChanged((user) => {
      if (user) {
        onUserLogin()
      }
    })
  }, [])
```